dotnet/src/SemanticKernel.Core/Contents/StreamingMethodContent.cs

```diff
@@ -23,13 +24,15 @@ public override byte[] ToByteArray()
         }
 
         // By default if a native value is not Byte[] we output the UTF8 string representation of the value
-        return Encoding.UTF8.GetBytes(this.Content?.ToString());
+        return this.Content?.ToString() is string s ?
+            Encoding.UTF8.GetBytes(s) :
+            Array.Empty<byte>();
     }
 
     /// <inheritdoc/>
     public override string ToString()
     {
-        return this.Content.ToString();
+        return this.Content.ToString() ?? string.Empty;
```

Comment:
Should we change the return type from `string` to `string?` since the base method defined in `Object` is using `string?`

https://github.com/dotnet/runtime/blob/4dffd80c4d77c27e772a0be26e8036af77fbb26e/src/libraries/System.Private.CoreLib/src/System/Object.cs#L39

Reply:
No, we want this to be non-nullable.

Reply:
Thanks